google/cloud/ndb/model.py

```diff
@@ -604,6 +604,12 @@ def new_entity(key):
 
                 continue
 
+        if value is not None and isinstance(prop, BlobProperty):
```

Comment:
I wonder if instead of special casing this here in ``_entity_from_ds_entity``, there should be some generalized way for properties to take into account (or not) the meaning field when deserializing themselves. I just added a ``_to_datastore`` method to ``Property`` in order to deal with #184, and I wonder if an overridable ``_from_datastore`` might be helpful here.